src/tools/roc_conv/main.cpp

```diff
@@ -68,23 +69,33 @@ int main(int argc, char** argv) {
 
     pipeline::ConverterConfig config;
 
-    if (args.frame_size_given) {
-        if (args.frame_size_arg <= 0) {
-            roc_log(LogError, "invalid --frame-size: should be > 0");
+    if (args.frame_length_given) {
+        if (!core::parse_duration(args.frame_length_arg, config.internal_frame_length)) {
+            roc_log(LogError, "invalid --frame-length: frame size unspecified");
```

Comment:
`frame size unspecified` -> `invalid format`

Reply:
The same in other tools.

Reply:
done.